cinema/views.py

```diff
@@ -1,6 +1,8 @@
+from django.db.models import F, Count
 from rest_framework import viewsets
+from rest_framework.pagination import PageNumberPagination
```

Comment:
Code Style issue: The imports are not grouped using parentheses `()`. This should be corrected to improve readability.